htdocs/accountancy/admin/account.php

```diff
@@ -336,11 +306,22 @@
     print "</select>";
     print ajax_combobox("chartofaccounts");
     print '<input type="submit" class="button" name="change_chart" id="change_chart" value="'.dol_escape_htmltag($langs->trans("ChangeAndLoad")).'">';
-    print '<input type="hidden" name="valid_change_chart" id="valid_change_chart" value="0">';
+
+    print '</form>';
 
     print '<br>';
 	print '<br>';
 
+	print '<form method="POST" id="searchFormList" action="'.$_SERVER["PHP_SELF"].'">';
```

Comment:
By splitting the form, you break a lot of feature, for example, you lose the page selected of the limit on list that must be into the form (the print_barre_liste contains input field for form)
A better solution would be to replace the button "Change And Load" that has type "submit" with a simple "<a>" link and add javascript onClick on it to make a simple javascript redirect of the request using the id of chart of account from the select box as parameter (this is the only parameter we need).

Reply:
@eldy 
and if i add the same var of other form ? look my last fix